client/src/components/FrontPage.js

```diff
@@ -12,9 +13,14 @@ const FrontPage = (props) => {
     )
   } else {
     checkLogin = (
-      <p>
-        Here are some puzzles!
-      </p>
+      <div>
+        <p>
+          Here are some puzzles!
+        </p>
+        <Link to="/puzzle/1"><button className="button">Easy Puzzle</button></Link>
+        <Link to="/puzzle/2"><button className="button">Medium Puzzle</button></Link>
+        <Link to="/puzzle/3"><button className="button">Hard Puzzle</button></Link>
+      </div>
     )
```

Comment:
This is fine to start!

but try to only make PRs for fully complete features. Realistically in your site, you probably wont just leave links to static links, but instead will have different lists of puzzles based on difficulty. Again though, this is okay to start to move forward and refactor later.

It is antipattern to nest a button in an anchor tag. `Link` actually generates an anchor tag on the DOM. Would you more likely want to do is pass styling to your link that makes it LOOK like a button, rather than use a button. Plus this might make the "click-zone" for the link a bit odd.